src/lib.rs

```diff
@@ -37,6 +37,7 @@ extern crate rustc_session;
 extern crate rustc_span;
 extern crate rustc_target;
 
+mod c_ffi_support;
```

Comment:
In line with my other commants, I think this should be called "ffi_support". If someone wants to write those .so files in Rust, they can. :)